src/main/jbake/content/connectorexample003.adoc

```diff
@@ -57,8 +57,8 @@ a traffic update looks like this:
  ]}
 ----
 
-The `traffic-rar` module implements the inbound contract of the Jakarta EE
-Connector Architecture. This module subscribes to the traffic
+The `traffic-rar` module implements Jakarta Connectors the inbound contract.
```

Comment:
Should be:

The `traffic-rar` module implements the inbound contract of the Jakarta Connectors specification.